### ISSUE TITLE
anitya doesn't run checks for new versions

### ISSUE THREAD

**Issue by ignatenkobrain** [release-monitoring/anitya#605]
We've noticed that we don't get notifications for new versions for projects with `crates.io` backend. They just doesn't seem to be checked.

Manual refresh does the trick.

**Comment by Heidistein** [comment]
I have noticed this indied on multiple projects. Django, apache httpd, libzip (was broken, and some more.
Today I noticed it with PHP

**Comment by Zlopez** [comment]
I will look into it.

**Comment by Zlopez** [comment]
According to the cron job log we got an unhandled exception:
```
Unhandled exception in thread started by <bound method
DummyProcess.__bootstrap of <DummyProcess(Thread-7, stopped daemon
140222260033280)>>
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 785, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib64/python2.7/threading.py", line 825, in __bootstrap_inner
    (self.name, _format_exc()))
  File "/usr/lib64/python2.7/traceback.py", line 240, in format_exc
    etype, value, tb = sys.exc_info()
AttributeError: 'NoneType' object has no attribute 'exc_info'
```

I will try to find a root of it, but in the first glance it looks like something is missing when creating process.

**Comment by Zlopez** [comment]
@srakitnican 
This is probably caused by the failing cron job.

**Comment by Zlopez** [comment]
It looks like the current version deployed on staging doesn't have this issue.
So after moving this to production, this should be solved.

**Comment by Zlopez** [comment]
When I tried to run cron job locally, I got `UnicodeEncodeError` caused by this project https://release-monitoring.org/project/16189/

It looks like I found the root of this issue.

I'm working on the fix now.

**Comment by Zlopez** [comment]
Should be fixed in 0.13.1
